.github/workflows/node.js.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deployment to AWS lambda
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  serverless-deploy:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install serverless
+      run: npm install -g serverless
+   # - run: npm run build --if-present
```

Comment:
Do you mind adding an `Install dependencies` as well here where we can do an `npm install` for the rest of the dependencies?

Reply:
Yes for sure, again same comment as previous depends if we will be using the separate Action for testing. This PR was purely for lambda function deployment as i mentioned in the "Purpose of the PR"

Reply:
Makes sense, this step can be added when needed